### PR TITLE
clears language in index_codec, index_matroska, index_tags

### PR DESCRIPTION
**Pull request by ablwr**
Following up on @robUx4's work in splitting, some typo clarifications, backticking, and replacing second-person ("you") language.

A separate PR will come along for tagging-video-example, tagging-audio-example.

### CODE REVIEW

File: index_codec.md [hunk 1]
```diff
@@ -33,23 +33,23 @@
...
+This document intends to define this mapping for many commonly used codecs in Matroska.
 
 # Status of this document
 
 This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concept already defined in the Matroska specifications defined by this workgroup.
 
 # Security Considerations
 
-Matroska Codecs inherits security considerations from EBML and Matroska.
+Matroska Codecs inherit security considerations from EBML and Matroska.
```
Comment (mbunkus) [inline index_codec.md]: I just realized that "Matroska Codecs" is supposed to refer to the title of this document. Therefore "inherits" would be correct. Indeed, the title attribute at the very top is "Matroska Codec" (singular), something that I'd like to see changed to the plural.

As to this line (and similar lines in the other two documents): maybe changing it to "This document inherits… from the documents…" would make it sound less confusing.

File: index_matroska.md [hunk 2]
```diff
@@ -174,9 +174,13 @@ For (possibly) Laced Data
...
+Lacing is a mechanism to save space when storing data. It is typically used for small blocks of data (referred to as frames in Matroska). There are 3 types of lacing:
 
-Let's say you want to store 3 frames of the same track. The first frame is 800 octets long, the second is 500 octets long and the third is 1000 octets long. As these data are small, you can store them in a lace to save space. They will then be solved in the same block as follows:
+1. Xiph, inspired by what is found in the Ogg container
+2. EBML, which is the same with sizes coded differently
+3. fixed-size, where the size is not coded
+
+For example, a user wants to store 3 frames of the same track. The first frame is 800 octets long, the second is 500 octets long and the third is 1000 octets long. As these data are small, they can be stored in a lace to save space. They will then be solved in the same block as follows:
```
Comment (mbunkus) [inline index_matroska.md]: "solved in the same block" sounds wrong. Maybe "stored…"?

File: index_matroska.md [hunk 3]
```diff
@@ -223,15 +227,15 @@ In this case only the number of frames in the lace is saved, the size of each fr
 
 #### SimpleBlock Structure
 
-The SimpleBlock is very inspired by the [Block structure](#block-structure). The main differences are the added Keyframe flag and Discardable flag. Otherwise everything is the same.
+The `SimpleBlock` is very inspired by the [Block structure](#block-structure). The main differences are the added Keyframe flag and Discardable flag. Otherwise everything is the same.
```
Comment (mbunkus) [inline index_matroska.md]: "is very inspired" ← remove "very", please.

File: index_tags.md [hunk 4]
```diff
@@ -39,17 +39,17 @@ This document defines the Matroska tags, namely the tag names and their respecti
...
 
 # Status of this document
 
-This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concept already defined in the Matroska specifications defined by this workgroup.
+This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concepts already defined in the Matroska specifications defined by this workgroup.
 
 # Security Considerations
 
-Tag values can be either strings or binary blobs. Matroska Codecs inherits security considerations from EBML and Matroska.
+`Tag` values can be either strings or binary blobs. Matroska `Tag Elements` inherits security considerations from EBML and Matroska.
```
Comment (mbunkus) [inline index_tags.md]: Same concern as above regarding "Matroska Codecs inherits…"

File: index_matroska.md (hunk 3 above)
Comment (ablwr) [inline index_matroska.md]: I thought about that too!

**Comment by ablwr** [comment]
interactively rebased, squashed, forced pushed 😎

File: index_codec.md [hunk 5]
```diff
@@ -33,23 +33,23 @@
...
+This document intends to define this mapping for many commonly used codecs in Matroska.
 
 # Status of this document
 
 This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concept already defined in the Matroska specifications defined by this workgroup.
 
 # Security Considerations
 
-Matroska Codecs inherits security considerations from EBML and Matroska.
+This document inherit security considerations from the EBML and Matroska documents.
```
Comment (mbunkus) [inline index_codec.md]: Well, as document is singular, "inherit" should really end with…? 😁

File: index_tags.md [hunk 6]
```diff
@@ -39,17 +39,17 @@ This document defines the Matroska tags, namely the tag names and their respecti
...
 
 # Status of this document
 
-This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concept already defined in the Matroska specifications defined by this workgroup.
+This document is a work-in-progress specification defining the Matroska file format as part of the [IETF Cellar working group](https://datatracker.ietf.org/wg/cellar/charter/). It uses basic elements and concepts already defined in the Matroska specifications defined by this workgroup.
 
 # Security Considerations
 
-Tag values can be either strings or binary blobs. Matroska Codecs inherits security considerations from EBML and Matroska.
+`Tag` values can be either strings or binary blobs. This document inherit security considerations from the EBML and Matroska documents.
```
Comment (mbunkus) [inline index_tags.md]: "inherits" again

**Comment by ablwr** [comment]
Silly silly copy-and-pasting me...!

**Pull request merged**